.golangci.yml

```diff
@@ -17,6 +17,11 @@ issues:
         - gosec
 
 linters-settings:
+  govet:
+    enable-all: true
+    disable:
+      - fieldalignment
+      - shadow
```

Comment:
Why do we need to add it?
The code is passing in the tests
What error are you facing that motivates you to add it?

Reply:
these 3 govet rules, fieldalignment, shadow and nilness, are not verified in the code. The other rules are

this PR fixes nilness rule and enforces it